src/rewards/PolygonID/components/ConnectDIDModal.tsx

```diff
@@ -23,18 +23,19 @@ import { useEffect } from "react"
 import useSWR from "swr"
 import useSWRImmutable from "swr/immutable"
 import useConnectedDID from "../hooks/useConnectedDID"
-import { useMintPolygonIDProofContext } from "./MintPolygonIDProofProvider"
 
 type Props = {
   isOpen: boolean
   onClose: () => void
+  onMintPolygonIDProofModalOpen: () => void
 }
 
-const ConnectDIDModal = ({ isOpen, onClose }: Props) => {
+const ConnectDIDModal = ({
+  isOpen,
+  onClose,
+  onMintPolygonIDProofModalOpen,
```

Comment:
What was the root of the issue here? We used a context so we don't need to pass down props like this, so it would be neat if we could solve this in some other way :D

Reply:
This component isn't used anywhere, but in that provider. Other solution perhaps could be if we moved this into the provider.

Reply:
> This component isn't used anywhere, but in that provider.
Oh, I missed that during the review, then it's great this way!